build.sh

```diff
@@ -16,83 +16,82 @@
 
 # Package build scripts can use this by setting $DESTDIR before launching the
 # script.
-prefix=/opt/google-cloud-ops-agent
+prefix=/usr/libexec/google-cloud-ops-agent
 sysconfdir=/etc
 systemdsystemunitdir=$(pkg-config systemd --variable=systemdsystemunitdir)
 systemdsystempresetdir=$(pkg-config systemd --variable=systemdsystempresetdir)
-subagentdir=$prefix/subagents
+# subagentdir=$prefix/subagents
 
 . VERSION
 if [ -z "$BUILD_DISTRO" ]; then
   release_version="$(lsb_release -rs)" #e.g. 9.13 for debian, 8.3.2011 for centos
   BUILD_DISTRO=$(lsb_release -is | tr A-Z a-z)${release_version%%.}
 fi
 if [ -z "$CODE_VERSION" ]; then
-  CODE_VERSION=${PKG_VERSION}
+  CODE_VERSION="$PKG_VERSION"
```

Comment:
We probably want to keep the curly braces as well…

Reply:
this PR adopts google bash style guide, which says no curly braces if the variable is by itself, always curly braces otherwise.